tools/wpt/expectation.json

```diff
@@ -14524,5 +14524,63 @@
       "performance.clearResourceTimings in workers",
       "performance.setResourceTimingBufferSize in workers"
     ]
+  },
+  "urlpattern": {
+    "urlpattern.any.html": [
+      "Pattern: [{\"pathname\":\"/foo/bar\"}] Inputs: [{\"pathname\":\"foo/bar\"}]",
```

Comment:
All failing tests are attributable to an upstream spec bug: https://github.com/denoland/rust-urlpattern/issues/8